src/components/filter.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+enum FilterType {
+  FILTERED_ACTIVE = 'Active',
+  FILTERED_COMPLETED = 'Completed',
+  CLEAR_COMPLETED = 'Clear completed',
+  UNFILTERED = 'All',
+}
+//
+type Props = {
+  errorMessage: string;
+  todos: Todo[];
+  todoCount: number;
+  setFilterBy: React.Dispatch<React.SetStateAction<FilterType>>;
+  handleClearCompleted: (completed: Todo[]) => void;
+  completedTasks: Todo[];
+};
+
+export const Filter: React.FC<Props> = ({
+  errorMessage,
+  todos,
+  todoCount,
+  setFilterBy,
+  handleClearCompleted,
+  completedTasks,
+}) => {
+  return (
+    <>
+      {!errorMessage && todos.length > 0 && (
```

Comment:
```suggestion
      {todos.length > 0 && (
```
But you can add this condition rendering in App.tsx - to render or not the entire component

Reply:
Still not fixed.